package.json

```diff
@@ -6,15 +6,13 @@
   "scripts": {
     "generate-gql-types": "apollo-codegen generate \"src/**/*.ts\" --schema \"src/types/github-schema.json\" --target typescript --addTypename --output \"src/types/gql-types.ts\"",
     "clean": "rm -rf ./build/",
-    "post-build": "node build.js",
-    "build:cli": "tsc -p src/cli/tsconfig.json --outDir build",
     "build:client": "tsc -p src/client/tsconfig.json --outDir build",
-    "build:server": "tsc -p src/server/tsconfig.json --outDir build",
-    "build": "npm run clean && npm run generate-gql-types && npm run build:client && npm run build:cli && npm run build:server && npm run post-build && npm run lint",
-    "build:watch": "npm run build && (npm run build:server -- --watch & npm run build:client -- --watch & npm run build:cli -- --watch)",
-    "deploy": "npm run build && rm -rf ./node_modules/ && npm install --only=production && npm run deploy:gcloud -- --project github-health-staging && npm install",
-    "deploy:prod": "npm run build && rm -rf ./node_modules/ && npm install --only=production && npm run deploy:gcloud -- --project github-health && npm install",
-    "deploy:gcloud": "gcloud app deploy app.yaml --quiet",
+    "build:server": "tsc -p src/tsconfig.json --outDir build",
+    "build": "npm run clean && npm run generate-gql-types && npm run build:client && npm run build:server && node copy-static-assets.js && cp {package.json,package-lock.json,app.yaml,secrets.json,github-health-*.json} build/ && npm run lint",
+    "build:watch": "npm run build && (npm run build:server -- --watch & npm run build:client -- --watch)",
+    "deploy:staging": "npm run build && cd build && gcloud app deploy app.yaml --quiet --project github-health-staging",
+    "deploy:prod": "npm run build && cd build && gcloud app deploy app.yaml --quiet --project github-health",
+    "launch": "npm run build && node build/server.js",
```

Comment:
`start` is usually more typical

Reply:
Hmm, GAE at some point will respect `start` right? In which case, this would be totally wrong. Maybe I'll just remove it for now.